docs/userguide.rst

```diff
@@ -96,6 +96,21 @@ so that a hard copy of the link target is uploaded to the cloud
 storage workspace. The link is not maintained throughout the
 workflow execution.
 
+Overriding default input parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to run a workflow with different inpute parameters than you have in
+ ``reana.yaml``, you can do it by running `reana-client start` with flag ``-p``
```

Comment:
There is a space at the beginning of this line and all the following ones which forces the render of a `tab` when Sphinx displays the docs.